google/cloud/tasks/v2beta3/cloudtasks.proto

```diff
@@ -357,7 +384,10 @@ message CreateQueueRequest {
   // The list of allowed locations can be obtained by calling Cloud
   // Tasks' implementation of
   // [ListLocations][google.cloud.location.Locations.ListLocations].
-  string parent = 1;
+  string parent = 1 [
```

Comment:
`Queue queue = 2` below in this message should also be required, according to the old GAPIC yaml